GameData/RealismOverhaul/RemoteTech_Settings.cfg

```diff
@@ -1,9 +1,14 @@
-RemoteTechSettings:NEEDS[RealismOverhaul]
+//  ==================================================
+//  Custom RemoteTech settings for Realism Overhaul.
+
+//  *   Make "Root" the default communication rule set.
+//  *   Allow combining multiple antennae for range extension.
+//  *   Zero the active engine throttle on connection loss.
```

Comment:
Took the chance to also change the MM patching modifiers since these fields are always going to be present.